### ISSUE TITLE
over 90% accuracy with pytorch FC

### ISSUE THREAD

**Issue by albertbn** [moritzhambach/machine-learning#1]
Hello Moritz, 

this is not an issue - I just haven't found another way to contact you,

First of all your numpy implementation of the **MNIST handwritten** digits has helped me immensely - in terms of the **weight initialization** and the update of the gradients with the learning rate and regularization param. I obviously had the last one implemented wrong.  Anyway, your implementation (based as I understand on Andrew Ng's course) helped me achieve great **validation accuracy - about 99%**. 
I have refactored the code and made it shorter and more generic - I can share the project if interested 

I've tried this model also on the **Fashion MNIST** datasets and it also achieves good results. 

I was wondering if you have any ideas as of how to implement a similar simple winning architecture with **pytorch**, just using **fully connected** layers (possibly with dropout and/or batch norm) 

I am able to somehow pass 90% accuracy on the validation set but with immense efforts - running first and **Adam** optimizer, then **SGD**... have tried different architecture settings + hyper params.

Thanks in advance,
Albert

thanks in advance,
Albert
